conans/client/conf/config_installer.py

```diff
@@ -86,6 +86,45 @@ def _filecopy(src, filename, dst):
     shutil.copyfile(src, dst)
 
 
+def _process_file(directory, filename, config, cache, output, folder):
+    if filename == "settings.yml":
+        output.info("Installing settings.yml")
+        _filecopy(directory, filename, cache.cache_folder)
+    elif filename == "conan.conf":
+        output.info("Processing conan.conf")
+        _handle_conan_conf(cache.config, os.path.join(directory, filename))
+    elif filename == "remotes.txt":
+        output.info("Defining remotes from remotes.txt")
+        _handle_remotes(cache, os.path.join(directory, filename))
+    elif filename in ("registry.txt", "registry.json"):
+        try:
+            os.remove(cache.remotes_path)
+        except OSError:
+            pass
+        finally:
+            _filecopy(directory, filename, cache.cache_folder)
+            migrate_registry_file(cache, output)
+    elif filename == "remotes.json":
+        # Fix for Conan 2.0
+        raise ConanException("remotes.json install is not supported yet. Use 'remotes.txt'")
+    # TODO : do we need this ? in which case we need to copy all the folder source
```

Comment:
yes, please leave this as-is, it is a feature to copy any other file in the folder, not just the known ones. For example, it is possible to install python files and use them in recipes (not recommended, legacy, but possible)

Reply:
OK. I was not sure of the usage, and, in the issue, the user want no necessary file copy.